stores/json/store.go

```diff
@@ -149,14 +149,24 @@ func (store Store) treeBranchFromJSONDecoder(dec *json.Decoder) (sops.TreeBranch
 	}
 }
 
+// Encoder to disable escaping html symbols
+// See: https://github.com/getsops/sops/issues/881
+func jsonMarshal(v interface{}) ([]byte, error) {
+	buffer := &bytes.Buffer{}
+	encoder := json.NewEncoder(buffer)
+	encoder.SetEscapeHTML(false)
+	err := encoder.Encode(v)
```

Comment:
`Encode` appends a newline (https://pkg.go.dev/encoding/json#Encoder.Encode), which `json.Marshal` apparently does not do. (This makes the unit tests fail.) This is apparently because `Encoder` is for JSON streams, while `Marshal` produces a single JSON file.

Reply:
Simply adding code which removes a trailing newline (if exists) probably suffices to fix this. I don't see another way to do this with Go's API.

Reply:
Shouldn't we stick with newline in new files instead?

Reply:
This function is called from `encodeValue` and used during serialization of any JSON value, so if you use this to create JSON of a map, you get someting like
```json
{
  "foo"
: "bar"
,
  "baz"
: 42
,
  "bam"
: null
}
```